### DISCUSSION TITLE
using loras in wildcards

### DISCUSSION THREAD

**Discussion by lgwjames** [lllyasviel/Fooocus#2287] [Q&A]
Just want to check I'm doing this right

Someone made me a lora from 30 photos for my ai girl and want to make sure that fooocus will read it correctly

do i do this

prompt 

__jessicaL__ , ((this should read the wildcard file correct? with about her details )) 
now if i want to kick in the lora, should i add just the keyword JessicaL inside that text file? as this is what was set as the JessicaL.safetensors

or do i have to add in prompt <lora: name: weight> so <lora: JessicaL: 0.8>

**Comment by poor7** [comment]
If I understand everything correctly, then neither one nor the other. Just turn on the LoRA and set its weight to the same place where you turned it on.

**Reply by lgwjames** [reply]
want to check, as like any loras you download, it says there is a key word to get it to work, so i was thinking do i need both words now or just the lora? and key word. so the lora should already know what my girl looks like?

**Reply by mashb1t** [reply]
you need to select the LoRA to activate it and, paraphrased, a keyword to focus the image generation process into the right direction of the pretrained LoRAs content.
In Fooocus, LoRAs are activated by selecting them until https://github.com/lllyasviel/Fooocus/pull/2210 is merged, then you can also only click a checkbox.
Depending on their training data they are more or less likely to resemble the subject of their training data.
